src/topology/compacts.lean

```diff
@@ -234,4 +239,118 @@ let ⟨s, hs⟩ := exists_compact_subset is_open_univ $ mem_univ (classical.arbi
   ⟨{ carrier := s, compact' := hs.1, interior_nonempty' := ⟨_, hs.2.1⟩ }⟩
 
 end positive_compacts
+
+/-! ### Clopen sets -/
+
+/-- The type of clopen sets of a topological space. -/
+structure clopens (α : Type*) [topological_space α] :=
+(carrier : set α)
+(clopen' : is_clopen carrier)
```

Comment:
Should this (and its API) move to a different file? I wouldn't expect it in `compacts.lean`.

Reply:
I agree that this doesn't match the file name, but `closeds` is in this file already. I was thinking that we could either rename the file to `topology.sets` or`split it into `topology.sets.closeds`, `topology.sets.compacts` and rename `topology.opens to `topology.sets.opens`.

Do you want me to do this?